pkg/controller/volume/attachdetach/cache/actual_state_of_world_test.go

```diff
@@ -26,6 +26,36 @@ import (
 	volumetesting "k8s.io/kubernetes/pkg/volume/testing"
 )
 
+func verifyAttachedVolume(
```

Comment:
nit: Why move the helper method to the top of the file? I'd prefer to leave the helper method near the bottom where it was (makes the diff easier too).